docs/source/_templates/layout.html

```diff
@@ -9,6 +9,24 @@
   }}();
 </script> -->
 
+<title>EvalML - Alteryx Open Source AutoML Library</title>
+<meta name="description" content="EvalML is an AutoML library that builds, optimizes, and evaluates machine learning pipelines using domain-specific objective functions. — EvalML 0.17.0 documentation">
```

Comment:
Are we leaving the version at 0.17 or updating to 0.18?

Reply:
Oh good catch! I'll just delete the version altogether so it doesn't have to be constantly updated :P